cluster/gce/gci/configure-helper.sh

```diff
@@ -1561,7 +1564,7 @@ if [[ "${KUBERNETES_MASTER:-}" == "true" ]]; then
   create-master-etcd-auth
 else
   create-node-pki
-  create-kubelet-kubeconfig
+  create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
```

Comment:
The GCE tests should cover the "is registered" case and the GKE tests should cover the "is not registered" case.